services-custom/dynamodb-enhanced/README.md

```diff
@@ -184,18 +206,20 @@ that write will fail.
 To load the extension:
 ```java
 MappedDatabase database = 
-  MappedDatabase.builder()
-                .dynamoDbClient(dynbamoDbClient)
-                .extendWith(VersionedRecordExtension.builder().build())
-                .build();
+  DynamoDbMappedDatabase.builder()
+                        .dynamoDbClient(dynbamoDbClient)
+                        .extendWith(VersionedRecordExtension.builder().build())
```

Comment:
Is there a create method on VersionedRecordExtension if it can be done no-args?

Reply:
There probably should be, but there isn't, so this is accurate. We plan to integrate that plugin into the core of the mapper anyway so it will be moot soon.